packages/salesforcedx-vscode-apex/test/index.ts

```diff
@@ -13,7 +13,8 @@ const testRunner = require('@salesforce/salesforcedx-utils-vscode/out/src/test/t
 testRunner.configure({
   ui: 'bdd', // the TDD UI is being used in extension.test.ts (suite, test, etc.)
   useColors: true, // colored output from test results
-  timeout: 360000
+  timeout: 360000,
+  retries: 2
```

Comment:
This is not a bad idea. We might want to consider adding this to other tests as well.

CC: @lcampos, @DatGuyJonathan - I don't think James Sweetman got around to implementing the simple "re-run" strategy from https://www.jonbell.net/icse18-deflaker.pdf